app/controllers/api_controller/renderer.rb

```diff
@@ -457,7 +457,7 @@ def fetch_typed_subcollection_actions(method, is_subcollection)
       return unless is_subcollection
       ctype = @req[:collection].to_sym
       sakey = "#{@req[:subcollection]}_subcollection_actions".to_sym
-      collection_config.fetch_path(ctype, sakey, method.to_sym)
+      collection_config[ctype][sakey] && collection_config[ctype][sakey][method.to_sym]
```

Comment:
There might be other ways,  @Fryguy any recommendation on getting our beloved fetch_path() in Settings ?